jsk_perception/src/apply_mask_image.cpp

```diff
@@ -113,12 +125,22 @@ namespace jsk_perception
       cv::bitwise_not(mask, mask);
     }
 
+    cv::Rect region = jsk_recognition_utils::boundingRectOfMaskImage(mask);
+    if (camera_info_){
+      sensor_msgs::CameraInfo roi(*camera_info_);
```

Comment:
change name from `roi` to `camera_info`.

Reply:
OK, I will change it.
For reference, attention_clipper also used the name roi.
https://github.com/jsk-ros-pkg/jsk_recognition/blob/81b136d28dfa6f036e55125cb3e2861093123cce/jsk_pcl_ros/src/attention_clipper_nodelet.cpp#L361

Reply:
we should make code better.